client/src/i18n/en-US/index.js

```diff
@@ -509,8 +509,14 @@ export default {
   submissions: {
     details_heading: "Submission Details",
     action: {
-      review: "Review Submission",
-      change_status: "Change Status",
+      review: {
+        name: "Review Submission",
+        no_access: "Reviewers can only access submissions in reviewable states",
+      },
+      change_status: {
+        name: "Change Status",
+        no_access: "Cannot change the status of submission.",
```

Comment:
```suggestion
        no_access: "Rejected submissions cannot have their statuses changed.",
```

Since we've decided for now that changing a submission's status will be unavailable for rejected submissions, the tooltip message should communicate this clearly.